packages/devextreme-react/src/accordion.ts

```diff
@@ -124,25 +124,28 @@ type IItemProps = React.PropsWithChildren<{
   titleRender?: (...params: any) => React.ReactNode;
   titleComponent?: React.ComponentType<any>;
 }>
-const _componentItem = memo(
-  (props: IItemProps) => {
-    return React.createElement(NestedOption<IItemProps>, { ...props });
-  }
-);
-
-const Item: typeof _componentItem & IElementDescriptor = Object.assign(_componentItem, {
-  OptionName: "items",
-  IsCollectionItem: true,
-  TemplateProps: [{
-    tmplOption: "template",
-    render: "render",
-    component: "component"
-  }, {
-    tmplOption: "titleTemplate",
-    render: "titleRender",
-    component: "titleComponent"
-  }],
-})
+const _componentItem = (props: IItemProps) => {
+  return React.createElement(NestedOption<IItemProps>, {
+    ...props,
+    elementDescriptor: {
+      OptionName: "items",
+      IsCollectionItem: true,
+      TemplateProps: [{
+        tmplOption: "template",
+        render: "render",
+        component: "component"
+      }, {
+        tmplOption: "titleTemplate",
+        render: "titleRender",
+        component: "titleComponent"
+      }],
+    },
+  });
+};
+
+const Item = Object.assign<typeof _componentItem, NestedComponentMeta>(_componentItem, {
+  componentType: "option",
```

Comment:
```suggestion
  componentKind: "option",
```

There is always mess with the word `type` in metadata, need to be careful when using it to avoid misunderstanding.

We're using [TypeScript inspired](https://www.typescriptlang.org/docs/handbook/typescript-in-5-minutes-func.html#discriminated-unions
) word "kind" for such cases as this one.
A "type" is a TypeScript type and a "kind" is a specificator of a variation.

In this example `Shape` is a type (TypeScript-world type), and `'circle' | 'square' | 'triangle'` is a kind (variation specificator)
```
type Shape =
  | { kind: "circle"; radius: number }
  | { kind: "square"; x: number }
  | { kind: "triangle"; x: number; y: number };
```

Reply:
Can we please leave this rename to a follow-up PR to avoid the need to re-release `generators` right now?